device/mellanox/x86_64-nvidia_sn5640_simx-r0/ACS-SN5640/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,128 @@
+{#
+    Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+    Apache-2.0
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+#}
+{% set default_cable = '300m' %}
+{% set ingress_lossless_pool_size =  '31502336' %}
+{% set ingress_lossy_pool_size =  '31502336' %}
+{% set egress_lossless_pool_size =  '138240000' %}
```

Comment:
I am afraid this value is too bug. if the guaranteed size is 130.9, the number should be round_down(130.9*1024*1024, 192)